pkg/generator/ingress_translator.go

```diff
@@ -89,10 +95,31 @@ func (translator *IngressTranslator) translateIngress(ctx context.Context, ingre
 		}
 
 		secret, err := translator.secretGetter(ingressTLS.SecretNamespace, ingressTLS.SecretName)
-		if err != nil {
+		if apierrors.IsNotFound(err) {
```

Comment:
So we are trying to make the UX easier and we add the label ourselves.
Are there any other secrets that might need the same?

Reply:
Other secrets are "static" secret and we have already added the label like https://github.com/knative/serving/blob/main/config/core/300-secret.yaml#L34 by default. So we don't need it.